src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -62,6 +62,7 @@
 import hudson.slaves.NodeProperty;
 import hudson.util.DaemonThreadFactory;
 import hudson.util.Iterators;
+import hudson.util.LogTaskListener;
```

Comment:
Should be safely remotable since it delegates to a `StreamTaskListener` which in turn uses a `RemoteOutputStream` so that the original `LogOutputStream` need not be serialized. Should not matter for `JCloudsArtifactManager` anyway since its `clearAllStashes` does not need to remote the `TaskListener`: deletion happens from the master only.